manifest.js

```diff
@@ -0,0 +1,62 @@
+export default {
+  "_metadata": {
+    "major_version": 2
+  },
+  "display_information": {
+    "name": "reverse"
+  },
+  // Once Manifest APIs support this, we'll add it
+  // "runtime_environment": "slack",
```

Comment:
we may want to add a `runtime` or `bundled_runtime` field too

Reply:
Yah, I think we decided we don't need a `bundled_runtime`, but `runtime` is still needed. The deno builder actually sets this if it's missing currently (I think to `deno1.x`), but am open to if we really want that "magic" or not. I'll add `runtime` here for now just so it's explicit

Reply:
But this reminds me we had talked about somewhere, us doing a check or warning if you try to bundle with a version of deno that isn't "compatible" with your `runtime`. This probably belongs in `deno-slack-builder` since it's doing all of that.